frontend/app/src/components/inputs/AssetSelect.vue

```diff
@@ -118,18 +118,18 @@ watch(search, search => {
 
 watchThrottled(
```

Comment:
so for the watchThrottled, for example we type `BTC`
it runs twice for 
(search = "B", old = "") and
(search = "BTC", old = "BT")

so when it's the turn for `search=BTC` it tried to abort controller for `BT` which doesn't exist, instead it should abort controller for `B`